templates/managed/firefighting-support.html

```diff
@@ -0,0 +1,535 @@
+{% extends "managed/_base_managed-apps.html" %}
+
+{% block title %}Firefighting Support: enhanced cloud support from Canonical{% endblock %}
+
+{% block meta_description %}
+  Get enhanced cloud support from Canonical. Firefighting Support is an alternative to managed services that keeps your cloud or AI infrastructure running smoothly.
+{% endblock meta_description %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1sWNMmLoyOFUI9qZE3fqPhgZWJSpb-e6-juGrW7A5rr0/edit
+{% endblock meta_copydoc %}
+
+{% block body_class %}is-paper{% endblock body_class %}
+
+{% block content %}
+
+  <section class="p-strip is-shallow">
+    <div class="p-section">
+      <div class="row--50-50">
+        <div class="col">
+          <h1>Firefighting Support</h1>
+        </div>
+        <div class="col">
+          <p>
+            The best self-managed alternative to <a href="/managed">Canonical Managed Services</a>. Get engineers on call to help you resolve the most critical issues, while continuing to manage your own environments.
+          </p>
+          <hr class="is-muted" />
+          <a href="/managed/contact-us" class="p-button--positive js-invoke-modal">Enhance your cloud support now</a>
+        </div>
+      </div>
+    </div>
+    <div class="p-section">
+      <div class="u-fixed-width u-align--center">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/81320e0d-smart-city.png",
+          alt="",
+          width="916",
+          height="526",
+          hi_def=True,
+          loading="auto"
+          ) | safe
+        }}
```

Comment:
Are we not hiding such illustrations for smaller screens? They are pretty big and increase scrolling uncessesarily

Reply:
@camillet16